src/controllers/playback/video/index.html

```diff
@@ -66,6 +66,13 @@ <h3 class="osdTitle"></h3>
                     <span class="endsAtText"></span>
                 </div>
 
+
+                <button is="emby-ratingbutton" type="button" class="emby-button button-flat hide btnUserRating detailButton" title="${Rate}">
```

Comment:
This will fix the navigation in TV mode, but the focus indication will be different from the others.
```suggestion
                <button is="emby-ratingbutton" type="button" class="button-flat hide btnUserRating detailButton" title="${Rate}">
```